addons/account_edi/models/account_edi_document.py

```diff
@@ -265,12 +268,15 @@ def _process_documents_web_services(self, job_count=None, with_commit=True):
                     if attachments_potential_unlink:
                         self._cr.execute('SELECT * FROM ir_attachment WHERE id IN %s FOR UPDATE NOWAIT', [tuple(attachments_potential_unlink.ids)])
 
-                    self._process_job(documents, doc_type)
             except OperationalError as e:
                 if e.pgcode == '55P03':
                     _logger.debug('Another transaction already locked documents rows. Cannot process documents.')
+                    no_commit = True
+                    if not with_commit:
+                        raise UserError(_('This document is being sent by another process already. '))
```

Comment:
We should not process anything if we have that error (or any error)...
No need for the `no_commit` variable, and extra space at the end of the string

```suggestion
                    if not with_commit:
                        raise UserError(_('This document is being sent by another process already.'))
                    return
```

Also add it in the translation files

Reply:
@william-andre We are in a for loop however.  `continue` then?

Reply:
Yes